apps/src/lab2/views/components/TextToSpeech.module.scss

```diff
@@ -3,13 +3,27 @@
 
 .playButton {
   @include remove-button-styles;
+  display: flex;
   position: absolute;
-  top: 6px;
-  right: 6px;
-  opacity: 50%;
+  bottom: 7px;
+  right: 7px;
+  opacity: 35%;
   cursor: pointer;
   transition: opacity 0.2s ease;
 
+  html[dir="rtl"] & {
+    right: initial;
+    left: 6px;
```

Comment:
nit: I wonder if can stick with 7px here as well as for ltr languages?

Reply:
Good catch; done.